server/src/main/resources/applicationinsights.json

```diff
@@ -0,0 +1,7 @@
+{
+  "instrumentation": {
+    "jdbc": {
+      "enabled": false
```

Comment:
Without this, we send an event for every single sql statement, which is... a lot. I saw around 1k per minute from a single *idle* instance.

Reply:
👍 it seems like the RDBMS instance's own query log would be the right place to look for these if we needed them.